runtime/common/src/crowdloan.rs

```diff
@@ -254,14 +245,18 @@ decl_event! {
 
 decl_error! {
 	pub enum Error for Module<T: Config> {
-		/// The first slot needs to at least be less than 3 `max_value`.
-		FirstSlotTooFarInFuture,
-		/// Last slot must be greater than first slot.
-		LastSlotBeforeFirstSlot,
-		/// The last slot cannot be more then 3 slots after the first slot.
-		LastSlotTooFarInFuture,
+		/// The current lease period is more than the first lease period.
+		FirstPeriodInPast,
+		/// The first lease period needs to at least be less than 3 `max_value`.
+		FirstPeriodTooFarInFuture,
```

Comment:
This is just overflow?

Reply:
What do you mean here? That this error should be replaced with "Overflow" error?

Reply:
Yes. Otherwise the description is confusion and I was looking for some max_value constant in trait Config